SET_OF_STANDARDS.md

```diff
@@ -0,0 +1,50 @@
+# Programming Set of Standards
+
+## Purpose
+These are primarily guidelines on contributing ideas and code to the WeLike codebase. As well, there are other helpful insights into what is expected of us as technology industry professionals.
+
+## Note
+Since ppl can’t meet at the same time, please add your coding needs, NTKs, and suggestions below. Create sections as well. If there is a suggestion that you do NOT agree with, do NOT delete it! Instead add a comment to it and tag the contributor.
+
+Interesting source - JavaScript Coding Standards
+
+## Git
+Keywords for A Commits's Subject Matter Message: feat | fix | chore | refactor | docs | style | test | perf | ci | build | revert
+
+When in doubt, refer to this helpful freeCodeCamp post! [How to Level Up Your Git Commit Messages](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/)
+
```

Comment:
Many of the conventions and rules around committing come from this [post](https://cbea.ms/git-commit/) from Chris Beams, which outlines in detail the desired traits of a commit message. I've worked on teams that have installed pre-commit hooks so that one can't commit a bad message. It's a step to consider over time, I think.